### DISCUSSION TITLE
Setting Up GitHub Actions for V | The V Programming Language

### DISCUSSION THREAD

**Discussion by giscus** [vlang/vlang-blog#22] [General]
# Setting Up GitHub Actions for V | The V Programming Language

This article will walk you through setting up GitHub Actions for V. We begin with a primer on CI/CD, the various benefits and drawbacks. Then we make use of the Setup V action to set up a CI/CD environment for V, how to set it up, and how to use it. We will close with a link to a more sophisticated example.

https://blog.vlang.io/setting-up-github-actions-for-v/

**Comment by holder66** [comment]
A super tutorial, Subhomoy! It was very helpful for me. Keep up the good work!
